start.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+set -eux
+
+if [ -z $1 ]; then
+  echo "Usage: ./run.sh <username>"
+  exit 1
+fi
+
+USERNAME=$1
+INSTALLER_IMAGE="registry.svc.ci.openshift.org/openshift/origin-v4.0:installer"
+ANSIBLE_IMAGE="quay.io/vrutkovs/openshift-40-centos"
+PODMAN="sudo podman"
+
+if [ ! -f ./pull_secret.json ]; then
+  echo "Pull secret not found!"
+  exit 1
+fi
+
+if [ ! -d injected ]; then
+  echo "injected directory not found!"
+  exit 1
+fi
+
+echo "Removing existing artifacts"
+rm -rvf install-config.yml .openshift_install_state.json .openshift_install.log *.ign || true
+
+echo
+echo "Fetching installer"
+${PODMAN} pull ${INSTALLER_IMAGE}
+${PODMAN} run -rm -ti ${INSTALLER_IMAGE} version
+
+echo
+echo "Creating bootstrap.ign"
+${PODMAN} run --rm \
```

Comment:
I still need to add `--privileged` here and later `podman run` commands.

Reply:
Added `PODMAN_RUN` and `PODMAN_PULL` for this